packages/cli/src/utils/telemetry.ts

```diff
@@ -0,0 +1,91 @@
+import { ux } from '@oclif/core'
+import chalk from 'chalk'
+import packageJson from '../../package.json'
+import { CeloConfig } from './config'
+
+const debug = require('debug')('celocli:telemetry')
+
+export type TelemetryOptions = {
+  command: string
+  success: boolean
+  version: string
+}
+
+const getTelemetryOptions = (command: string, success: boolean): TelemetryOptions => {
+  return {
+    command,
+    success,
+    version: packageJson.version,
+  }
+}
+
+export const reportUsageStatisticsIfTelemetryEnabled = (
```

Comment:
seems like when there's a CLI error then it doesn't send any telemetry (it starts, but never sends) - to be investigated

Reply:
most likely the reason is because error is eventually handled by [this code](https://github.com/oclif/core/blob/efd8403dad32da8b5a3438b6cba214b9b5515d49/src/errors/handle.ts#L30) which calls `process.exit()` hence terminating any still running promises